public/app/features/explore/Explore.tsx

```diff
@@ -275,6 +277,9 @@ export class Explore extends React.PureComponent<ExploreProps> {
     const styles = getStyles();
     const StartPage = datasourceInstance?.components?.ExploreStartPage;
     const showStartPage = !queryResponse || queryResponse.state === LoadingState.NotStarted;
+    const queryErrors = queryResponse.error ? [queryResponse.error] : undefined;
+    const refId = getValueWithRefId(queryErrors);
+    const queryError = refId ? null : getFirstQueryErrorWithoutRefId(queryErrors);
```

Comment:
What does this mean? What kind of errors would not have a refID?

Reply:
From what I've seen, the generic (non-query-row-specific) ones don't have refID set up - e.g. when you look at datasource connection errors.

Reply:
Lets add a comment explaining what kind of errors they are then.